projects/rero/ng-core/src/lib/record/editor/toggle-wrapper/toggle-wrappers.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { FieldWrapper } from '@ngx-formly/core';
+import { isEmpty, removeEmptyValues } from '../utils';
+
+@Component({
+  selector: 'ng-core-editor-formly-toggle-wrapper',
+  template: `
+    <div class='toggle-wrapper'>
+      <div class="custom-control custom-switch">
+        <input class="custom-control-input" type="checkbox" id="toggle-switch" (change)="toggle($event)" [checked]="tsOptions.enabled">
+        <label class="custom-control-label" for="toggle-switch">{{ tsOptions.label }}</label>
+      </div>
+      <small class="form-text text-muted" *ngIf="tsOptions.description">{{ tsOptions.description }}</small>
+      <ng-container *ngIf="tsOptions.enabled" #fieldComponent></ng-container>
+    </div>
+  `
+})
+export class ToggleWrapperComponent extends FieldWrapper implements OnInit {
+
+  tsOptions = {
+    label: 'Toggle',
+    description: null,
+    enabled: false
+  };
+
+  ngOnInit() {
+    if (this.to['toggle-switch']) {
+      this.tsOptions = {...this.tsOptions, ...this.to['toggle-switch']};
+    }
+    this.tsOptions.enabled = !isEmpty(removeEmptyValues(this.model));
```

Comment:
Can you check that you do not have any error in your browser console? Just to be sure.

Reply:
Working if `toggle-switch` is present into jsonschema and if is not present.
 `!isEmpty(removeEmptyValues(this.model))` works if this.model contains 0 (enabled == false), 1 or 2 (enabled == true). 
I will push a WIP PR on rero-ils as soon as possible, to test it yourself ;)